opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -1,622 +1,179 @@
 #org.opennms.provisiond.enableDiscovery=true
-
-# Prior to 1.10 it was possible to delete entities that have been provisioned as
-# part of a provisioning group.  In 1.10 we have disabled this so that in order to 
-# delete these entities you have to go back to the provisioning group and delete them
-# from there.  To reenable this deletion you can set this to true.
-# NOTE: if you do this then the object will be recreated when the provisioning group is
-# next imported/synchronized
 #org.opennms.provisiond.enableDeletionOfRequisitionedEntities=false
-
-# This property is used to control the rescan scheduling existing nodes in
-# the database when Provisiond starts. The default value is true.
-# There are situations like distributed environments, where OpenNMS is deployed across
-# multiple servers, on which this feature must be disabled to avoid continuity issues.
-# In this scenario, most likely the inventory of nodes should not be managed by all OpenNMS instances.
 #org.opennms.provisiond.scheduleRescanForExistingNodes=true
-
-# Use this property to disable rescans of existing nodes following
-# an import (synchronize) of a provisioning group (requistion).  Default
-# behavior has always been true.
 #org.opennms.provisiond.scheduleRescanForUpdatedNodes=true
-
-# Use this property to change the strategy used for managing deployed/pending requisitions.
-# Tested strategies:
-# - file (default)
-# - fastFile (recommended for fastest response, at expense of memory resources)
-# Experimental strategies:
-# - fused
-# - fastFused
-# - queueing
-# - fastQueueing
-# - caching
-# - fastCaching
 #org.opennms.provisiond.repositoryImplementation=file
 
-###### MAPPING AND GEOCODING ######
+### Web UI
 
-# the map implementation to use
-# current choices are: GoogleMaps, Mapquest, OpenLayers
-gwt.maptype=OpenLayers
+#opennms.assets.allowHtmlFields = comments, description
+#org.opennms.web.aclsEnabled=false
+opennms.eventlist.acknowledge=false
+opennms.eventlist.showCount=false
+#opennms.nodesWithOutages.count=12
+#opennms.nodesWithOutages.show=true
+#opennms.nodesWithProblems.count=16
+#opennms.nodesWithProblems.show=true
+#opennms.nodeStatusBar.show=false
+#org.opennms.web.console.centerUrl=/surveillance-box.jsp
 
-# The API key to use for the remote monitor map
-gwt.apikey=
+### Dashboard and Surveillance View
+#org.opennms.dashboard.implementation=gwt
+#org.opennms.dashboard.redirect=false
 
-# The class options to use for geocoding.  Choices are:
+### Graphing
 
-# (Google maps API)
-#gwt.geocoder.class=org.opennms.features.poller.remote.gwt.server.geocoding.GoogleMapsGeocoder
+#org.opennms.web.graphs.engine=png
 
-# (Mapquest API)
-#gwt.geocoder.class=org.opennms.features.poller.remote.gwt.server.geocoding.MapquestGeocoder
+### Mapping and Geocoding
 
-# (OpenStreetMaps API)
+gwt.maptype=OpenLayers
+gwt.apikey=
+#gwt.geocoder.class=org.opennms.features.poller.remote.gwt.server.geocoding.GoogleMapsGeocoder
```

Comment:
The geocoder section needs an explanation that you are choosing one implementation among the 3 options (Google, Nominatim, Mapquest).